pkg/asset/cluster/azure/azure.go

```diff
@@ -0,0 +1,21 @@
+// Package azure extracts AZURE metadata from install configurations.
+package azure
+
+import (
+	"fmt"
+
+	"github.com/openshift/installer/pkg/types"
+	"github.com/openshift/installer/pkg/types/azure"
+)
+
+// Metadata converts an install configuration to Azure metadata.
+func Metadata(clusterID, infraID string, config *types.InstallConfig) *azure.Metadata {
+	return &azure.Metadata{
+		Region: config.Platform.Azure.Region,
+		Identifier: []map[string]string{{
+			fmt.Sprintf("kubernetes.io.cluster.%s", infraID): "owned",
+		}, {
+			"openshiftClusterID": clusterID,
```

Comment:
TODO: this might not be required.

Reply:
azure doesn't support `/` in tags, and I don't know what these tags are used for in openshift.
to get the terraform deployment to work, I changed the `/` with a `.`.
if the `kubernetes/io/cluster' tags  are not needed for openshift, I'm happy to remove them